packages/tsconfig-reference/scripts/tsconfigRules.ts

```diff
@@ -310,18 +310,40 @@ function formatAllowedValues(type: CommandLineOption["type"]) {
 }
 
 export const releaseToConfigsMap: { [key: string]: AnOption[] } = {
+  "5.7": ["rewriteRelativeImportExtensions"],
+  "5.6": ["strictBuiltinIteratorReturn", "noUncheckedSideEffectImports"],
+  "5.5": ["isolatedDeclarations", "noCheck"],
+  "5.0": [
```

Comment:
There really weren't any new flags in between? Wow.

Reply:
Yup - only formatting changes:
`git diff -U1000 v5.0.2 v5.4.5 src/compiler/types.ts`